plottr/apps/inspectr.py

```diff
@@ -529,7 +579,8 @@ def setRunSelection(self, runId: int) -> None:
         for k, v in structure.items():
             v.pop('values')
         contentInfo = {'Data structure': structure,
-                       'QCoDeS Snapshot': snap}
+                       'QCoDeS Snapshot': snap,
+                       'Metadata': ds.metadata}
```

Comment:
metadata is generalyl more useful than snapshot, hence shall it be viewed before snapshot?
```suggestion
                       'Metadata': ds.metadata,
                       'QCoDeS Snapshot': snap}
```